src/archivist/services/index.js

```diff
@@ -11,6 +12,10 @@ const fs = fsApi.promises;
 const __dirname = path.dirname(fileURLToPath(import.meta.url));
 const declarationsPath = path.resolve(__dirname, '../../..', config.get('services.declarationsPath'));
 
+const require = createRequire(import.meta.url);
+
+export const DOCUMENT_TYPES = require(path.resolve(__dirname, './documentTypes.json')); // eslint-disable-line import/no-dynamic-require
```

Comment:
Why make this a dynamic require?

```suggestion
export const DOCUMENT_TYPES = require(./documentTypes.json');
```

However, as outlined in [this article](https://www.stefanjudis.com/snippets/how-to-import-json-files-in-es-modules-node-js/#option-1%3A-read-and-parse-json-files-yourself), I'm in favour of using `JSON.parse` for readability reasons, since neither option feels right.

Reply:
Because I had to choose one, and I stuck to the method already chosen in the codebase. 
But I understand your point so I will migrate the other dynamic require to `JSON.parse`.